controller/rebuild.go

```diff
@@ -76,17 +91,29 @@ func (c *Controller) VerifyRebuildReplica(address string) error {
 		return err
 	}
 
-	logrus.Infof("chain %v from rw replica %s", rwChain, rwReplica.Address)
-	// Don't need to compare the volume head disk
-	rwChain = rwChain[1:]
-
 	chain, err := getReplicaChain(address)
 	if err != nil {
 		return err
 	}
+	WOcheckpoint, err := getReplicaCheckpoint(address)
+	if err != nil {
+		return err
+	}
+	var (
+		indx     int
+		snapshot string
+	)
+	for indx, snapshot = range rwChain {
+		if snapshot == WOcheckpoint {
```

Comment:
Should we check that WOcheckpoint should exist in rwChain if WOcheckpoint is not empty?